pkg/sink/codec/common/config.go

```diff
@@ -94,6 +98,8 @@ func NewConfig(protocol config.Protocol) *Config {
 		OnlyOutputUpdatedColumns:   false,
 		DeleteOnlyHandleKeyColumns: false,
 		LargeMessageHandle:         config.NewDefaultLargeMessageHandleConfig(),
+
+		TimeZone: time.Local,
```

Comment:
Will TimeZone be configurable in the future?

Reply:
It is reassigned to follow the tz in the config:

```go
	tz, err := util.GetTimezone(config.GetGlobalServerConfig().TZ)
	if err != nil {
		return nil, cerror.WrapError(cerror.ErrSinkInvalidConfig, err)
	}
	encoderConfig.TimeZone = tz
```